google-cloud-firestore/lib/google/cloud/firestore/field_value.rb

```diff
@@ -62,6 +63,33 @@ def type
           @type
         end
 
+        ##
+        # The values to change to an individual field in document data,
+        # depending on the type of change.
+        #
+        # @return [Array<Object>] The values.
+        #
+        # @example
+        #   require "google/cloud/firestore"
+        #
+        #   firestore = Google::Cloud::Firestore.new
+        #
+        #   # Get a document reference
+        #   nyc_ref = firestore.doc "cities/NYC"
+        #
+        #   array_union = Google::Cloud::Firestore::FieldValue.array_union(
+        #     1, 2, 3
+        #   )
+        #   array_union.type #=> :array_union
+        #   array_union.values #=> [1, 2, 3]
+        #
```

Comment:
The members of this class should not need to be part of the public interface. Exposing this will lock us into an API, while not actually improving the usability of the client.

Reply:
Marked private.